scrapers/ariba/ariba_driver.py

```diff
@@ -41,22 +43,24 @@ def is_logged_in(self):
         return len(login) == 0
 
     def login(self):
-        username_path = Path('username.key')
-        password_path = Path('password.key')
+        username_path = Path("username.key")
+        password_path = Path("password.key")
         if not username_path.exists() or not password_path.exists():
-            raise FileNotFoundError('username.key or password.key not found')
+            raise FileNotFoundError("username.key or password.key not found")
```

Comment:
For my learning purposes, is there a reason you replaced single quotes with double quotes?